internal/provider/resource_gitlab_runner_test.go

```diff
@@ -0,0 +1,121 @@
+package provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/xanzy/go-gitlab"
+)
+
+func TestAccGitlabRunner_basic(t *testing.T) {
+	testAccCheck(t)
+
+	group := testAccCreateGroups(t, 1)[0]
+	//The runner token is not populated on the return from the group create, so re-retrieve it to get the token.
+	group, _, err := testGitlabClient.Groups.GetGroup(group.ID, &gitlab.GetGroupOptions{})
+	if err != nil {
+		t.Fail()
+	}
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() {},
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testAccCheckManagedLicenseDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(`
+				resource "gitlab_runner" "this" {
+					token = "%s"
+					description = "Lorem Ipsum"
+				}
+				`, group.RunnersToken),
+			},
+			{
+				ResourceName:      "gitlab_runner.this",
+				ImportState:       true,
+				ImportStateVerify: true,
+				//These need to be ignored since they don't come back in the "get" command
+				ImportStateVerifyIgnore: []string{"authentication_token", "token"},
+			},
+			{
+				Config: fmt.Sprintf(`
+				resource "gitlab_runner" "this" {
+					token = "%s"
+					description = "Lorem Ipsum Dolor Sit Amet"
+				}
+				`, group.RunnersToken),
+			},
+			{
+				ResourceName:      "gitlab_runner.this",
+				ImportState:       true,
+				ImportStateVerify: true,
+				//These need to be ignored since they don't come back in the "get" command
+				ImportStateVerifyIgnore: []string{"authentication_token", "token"},
```

Comment:
Not 100% sure, but since the `token` is part of the config you probably don't have to ignore it. Could you verify please?

Reply:
@timofurrer  - I did verify, and it is required to ignore it. I suspect because it doesn't come back in the "Read" API (sensitive value below is obviously transient, so it's not a problem to post it): 

![image](https://user-images.githubusercontent.com/50960557/166167144-e05dd1ad-2ec8-4a34-97ea-ffd8b37c56ad.png)

Reply:
interesting! Thanks for the verification 🎉